packages/assets-controllers/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
```

Comment:
I'm experimenting with adding changelogs directly to PRs. One thing I noticed is that in order to know which PR number to put here, I had to first push up this PR, then add these entries, so we'll have to note that if/when we decide to stick with this process.